e2e-tests/tests/420-text-plugin.ts

```diff
@@ -43,252 +63,207 @@ Scenario('Add new line in plugin using Enter', async ({ I }) => {
+    addNewTextPlugin(I)
+    I.seeNumberOfElements(selectors.textEditor, initialTextPluginCount + 1)
     I.pressKey('Backspace')
     I.type('- Plain text')
     I.see('Plain text')
 
-    I.say('Focus the first text plugin')
-    I.pressKey('ArrowUp')
-    I.pressKey('ArrowUp')
-
+    focusTextPlugin(I, 'first')
     I.say('Create an ordered list')
     I.type('- Unordered list')
-    I.click('$plugin-toolbar-button-nummerierte-liste')
+    I.click(selectors.numberedListButton)
     I.see('Unordered list', 'ol')
 
-    I.say('Focus the second text plugin')
-    I.pressKey('ArrowDown')
-
+    focusTextPlugin(I, 'second')
     I.say('Merge the 2 text plugins by pressing Backspace')
-    // We need to duplicate the 'Backspace' command since only once doesn't have the expected effect
     I.pressKey('Backspace')
     I.pressKey('Backspace')
-    I.seeNumberOfElements('$plugin-text-editor', initialTextPluginCount)
+    I.seeNumberOfElements(selectors.textEditor, initialTextPluginCount)
   }
 )
 
 Scenario.todo('Merge with next plugin using Delete key')
 
 Scenario('Undo', async ({ I }) => {
-  I.amOnPage('/entity/create/Article/1377')
-
-  I.click('$add-new-plugin-row-button')
+  I.amOnPage(pageUrl)
 
+  addNewTextPlugin(I)
   I.pressKey('Backspace')
 
   I.type('Some text')
   I.see('Some text')
 
-  I.click('$editor-toolbar-undo')
-
+  I.click(selectors.toolbarUndoButton)
   I.dontSee('Some text')
 })
 
 Scenario('Undo using keyboard', async ({ I }) => {
-  I.amOnPage('/entity/create/Article/1377')
-
-  I.click('$add-new-plugin-row-button')
+  I.amOnPage(pageUrl)
 
+  addNewTextPlugin(I)
   I.pressKey('Backspace')
 
   I.type('Some text')
   I.see('Some text')
 
   I.pressKey(['CommandOrControl', 'Z'])
-
   I.dontSee('Some text')
 })
 
 Scenario('Redo', async ({ I }) => {
-  I.amOnPage('/entity/create/Article/1377')
-
-  I.click('$add-new-plugin-row-button')
+  I.amOnPage(pageUrl)
 
+  addNewTextPlugin(I)
   I.pressKey('Backspace')
 
   I.type('Some text')
-
   I.see('Some text')
 
-  I.click('$editor-toolbar-undo')
-
+  I.click(selectors.toolbarUndoButton)
   I.dontSee('Some text')
 
-  I.click('$editor-toolbar-redo')
-
+  I.click(selectors.toolbarRedoButton)
   I.see('Some text')
 })
 
 Scenario('Redo using keyboard', async ({ I }) => {
-  I.amOnPage('/entity/create/Article/1377')
-
-  I.click('$add-new-plugin-row-button')
+  I.amOnPage(pageUrl)
 
+  addNewTextPlugin(I)
   I.pressKey('Backspace')
 
   I.type('Some text')
-
   I.see('Some text')
 
   I.pressKey(['CommandOrControl', 'Z'])
-
   I.dontSee('Some text')
 
-  // ! For some reason, the first redo does not work. The second one does. If
-  // one puts a pause() here and runs the command only once through the
-  // interactive shell , it works just as fine as clicking the button.
-  // Therefore, I thought the Ctrl+Y was maybe happening too quickly after the
-  // Ctrl+Z, but even with I.wait(1) inbetween, two executions were needed.
```

Comment:
Please, please, don't remove comments like this. Whoever added it, had a good reason to do it. If nothing else, it points out to everyone that there is a reason for the double redo in the next two lines.

Reply:
Restored